src/components/Proposal/View/Categories/GrantProposalView.tsx

```diff
@@ -41,7 +43,14 @@ function GrantProposalView({ config }: Props) {
       <ProposalDescriptionItem title={t('page.proposal_view.grant.abstract_title')} body={abstract} />
       <ProposalDescriptionItem
         title={t('page.proposal_view.grant.size_title')}
-        body={`${intl.formatNumber(size)} USD`}
+        body={
+          paymentToken
+            ? t('page.proposal_view.grant.size_body', {
+                amount: intl.formatNumber(size, CURRENCY_FORMAT_OPTIONS as any),
```

Comment:
You can move `intl.formatNumber(...)` to a variable to be reused.
`const formattedAmount = intl.formatNumber(size, CURRENCY_FORMAT_OPTIONS as any)`

(Currency can also be applied to the formatter so it returns the value with the USD string, but we can check that later)